docs/index.rst

```diff
@@ -1,27 +1,9 @@
-sqlalchemy-redshift
-===================
+===============================================
+Welcome to sqlalchemy-redshift's documentation!
+===============================================
```

Comment:
I'd vote to leave out this heading entirely if the REAME.rst already has a suitable heading naming the project.